.env.example

```diff
@@ -1,16 +1,15 @@
 BUNDLE_ANALYZE=false
 ENVIRONMENT_NAME=local
 
+# The URL for the domain your app is hosted on (used for generating the urls needed for SEO)
+# If you deploy to Vercel or Netlify it is configured through the respective config files (`vercel.json` and `netlify.toml`)
+NEXT_PUBLIC_BASE_URL=http://localhost:3000/
```

Comment:
I think we need the netlify.toml and vercel.json files like the others to make sure this gets set during deployment.

I didn't add them in this morning as the base url didn't exist 🙈 

for your convenience: 
https://github.com/contentful/template-blog-webapp-nextjs/blob/main/netlify.toml
https://github.com/contentful/template-blog-webapp-nextjs/blob/main/vercel.json